### DISCUSSION TITLE
대학생 공모전 참가 팀 업데이트 후 월드 접속 오류

### DISCUSSION THREAD

**Discussion by diddntjd99** [naverz/zepeto-studio-kor#136] [5. Q&A]
안녕하세요,
대학교 공모전 참가한 학생입니다.
이번 업데이트 이후에 출시가 완료된 저희 월드에 접속해보면 흰 화면으로만 뜨고 접속이 안되는걸 볼 수 있는데 어떻게 해야할까요..

**Comment by cheolhui-lee** [comment]
안녕하세요.
현재 AWS 의 데이터베이스 응답속도가 비정상 적으로 느려져서 우회할 수 있는 방법을 찾고 있습니다.
제작에 불편을 드려 정말 죄송합니다.
